package.json

```diff
@@ -112,23 +112,24 @@
     "@commitlint/config-conventional": "13.2.0",
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "13.4.0",
-    "@types/fontfaceobserver": "2.1.0",
-    "@types/jest": "27.5.2",
-    "@types/node": "14.14.31",
-    "@types/react": "18.0.17",
-    "@types/react-dom": "18.0.6",
-    "@types/react-redux": "7.1.24",
-    "@types/react-router-dom": "5.3.3",
-    "@types/react-test-renderer": "18.0.0",
-    "@types/rimraf": "3.0.2",
-    "@types/shelljs": "0.8.11",
-    "@types/styled-components": "5.1.26",
-    "@types/testing-library__jest-dom": "5.14.5",
-    "@types/webpack": "5.28.0",
-    "@types/webpack-env": "1.18.0",
+    "@types/fontfaceobserver": "^2.1.0",
```

Comment:
Hmm. I am kind of against these carets on versions. I have seen so many problems over the 4 years due to these. Some authors publish a minor version of a package that conflicts with others and BOOM you have a bug that you can never predict until the template is installed. Happened so many times. Can we go back to the fixed versions?

Reply:
I agree with what you’re saying but think it’s a good idea to have the carets specifically for the definitely typed packages. these get updated frequently and contain only type definitions.

Reply:
Ok so, just for the typings